libs/eavmlib/src/gpio.erl

```diff
@@ -118,7 +98,7 @@ open() ->
 %%          `gpio:set_direction/3' it will always read as low.
 %% @end
 %%-----------------------------------------------------------------------------
--spec read(GPIO :: gpio(), GPIONum :: pin()) -> high | low.
+-spec read(GPIO :: gpio(), Pin :: pin()) -> high | low.
 read(GPIO, GPIONum) ->
```

Comment:
Should we change the parameter to match the spec?  Not strictly necessary, but I would prefer it (for readability)

Same 👇🏽